### DISCUSSION TITLE
카카오맵 시간 리플레쉬 오류의 건

### DISCUSSION THREAD

**Discussion by masstige** [changdoc/naver-vaccine-macro#214] [Q&A]
![image](https://user-images.githubusercontent.com/88412271/128107226-069991a8-cfc3-4124-8903-a00d94f94462.png)

자동 리플레쉬될 때, <left_count_by_coords>만 retry request 되는데,
브라우저 개발자모드에 waterfall order를 살펴보면 'https://stat.tiara.kakao.com/track?{papameter}'도 같이 호출되어야만 시간이 업데이트 되는데요.

실제 ui상 시간이 업데이트 되지 않아도 백그라운드에서는 <left_count_by_coords> 호출만으로 잘 동작되는 것 인가요?

**Comment by changdoc** [comment]
네 api 응답 값에서 잔여백신이 존재한다고 판단될 경우 해당 병원 예약 신청 페이지로 이동 하는 형태로 UI는 관련 없이 동작하도록 되어 있습니다.
